app/services/reports/querier.rb

```diff
@@ -17,12 +17,20 @@ def perform
       OrderDetailListTransformerFactory.instance(order_details).perform
     end
 
-    def order_details
+    def order_details(options = {})
```

Comment:
I think we'll need to address this downstream in the `nu` branch.  With that branch I believe we add a few extra `includes`.  It was handled before but rebasing with master may have broke something.